docs/common/DocsExample.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="docs-example">
+
+
+    <div class="code-toggle-button">
+      <button @click="toggleCodeVisibility">
+        <span>&lt;/&gt;</span>
+      </button>
+    </div>
+
+        <!-- Toggle code  -->
+    <div v-if="isCodeVisible" class="code-container">
```

Comment:
Also If we could change the `v-if` here to `v-show` as elements shown with v-show are mounted once and remain in the DOM.

Reply:
Hi @ozer550,

Thanks for the detailed feedback! I’ve made the changes based on your suggestions. Here's a summary of what I’ve done:

1. Replaced the toggle button with KIconButton: I've updated the button to use the new component.
2. Updated the toggle icon: The icon now switches between < > and chevronUp when toggling the code visibility.
3. Added tooltip: The tooltip for the toggle button now says "Show/Hide Code" for better user experience.
4. Changed v-if to v-show: I've updated this to ensure the element remains in the DOM after being toggled.

https://github.com/user-attachments/assets/59f5f64a-a7cc-446b-92fe-6a15d36f354c